README.md

```diff
@@ -9,7 +9,7 @@ Gradle plugin to create a java application based on Clean Architecture following
 
 Plugin Implementation  
 ===================
-To use the plugin you need Gradle version 5 or later, to start add the following section into your 
+To use the plugin you need Gradle version 5.6 or later, to start add the following section into your 
```

Comment:
Hola @gonchalo620  gracias por tu colaboracion, me gustaria saber por que concluiste que solo es soportado para version 5.6 de gradle en adelante.

Reply:
Con versiones de gradle inferiores de 5.6 no funcionan las tareas (solo funciona la tarea 1 que crea la estructura), esto debido a que solo a partir de la versión 5.6 sacaron la funcionalidad de gradle.properties que se está utilizando en el proyecto.
https://docs.gradle.org/5.6/release-notes.html#central-management-of-plugin-versions-with-settings-script

Con la versión 5.3 me salía error:

Reply:
Perfecto, asi es validado.